From 5b767d614ece454d6b5580c5b4368bc1031d3172 Mon Sep 17 00:00:00 2001
From: Xiaowen Liu <b37945@freescale.com>
Date: Tue, 26 Feb 2013 13:50:23 +0800
Subject: [PATCH 164/310] ENGR00215174 update camera hal to v2 and add usb
 camera for sabreauto.

update camera hal to v2 for sabreauto.
add usb camera as front camera for sabreauto.

Signed-off-by: Xiaowen Liu <b37945@freescale.com>
---
 sabreauto_6q/BoardConfig.mk | 5 +++++
 sabreauto_6q/init.rc        | 3 +++
 2 files changed, 8 insertions(+)

diff --git a/sabreauto_6q/BoardConfig.mk b/sabreauto_6q/BoardConfig.mk
index cf4bb0e..6e9409d 100644
--- a/sabreauto_6q/BoardConfig.mk
+++ b/sabreauto_6q/BoardConfig.mk
@@ -67,6 +67,11 @@ TARGET_USERIMAGES_SPARSE_EXT_DISABLED := true
 # atheros 3k BT
 BOARD_USE_AR3K_BLUETOOTH := true
 
+# camera hal v2
+IMX_CAMERA_HAL_V2 := true
+
+BOARD_HAVE_USB_CAMERA := true
+
 # define frame buffer count
 NUM_FRAMEBUFFER_SURFACE_BUFFERS := 3
 
diff --git a/sabreauto_6q/init.rc b/sabreauto_6q/init.rc
index 08c6e71..281970f 100644
--- a/sabreauto_6q/init.rc
+++ b/sabreauto_6q/init.rc
@@ -43,8 +43,11 @@ on boot
     #For landscape mode, orient is 0
     #For portrait mode, orient is 90
     #the android before honycomb are all in portrait mode
+    setprop camera.disable_zsl_mode 1
     setprop back_camera_name adv7180_decoder
     setprop back_camera_orient 0
+    setprop front_camera_name uvc
+    setprop front_camera_orient 0
 
     # Set OpenGLES version
     setprop ro.opengles.version 131072
-- 
1.8.0

